### DISCUSSION TITLE
Recipes for spekaer segmentation model

### DISCUSSION THREAD

**Discussion by vanIvan** [pyannote/pyannote-audio#1745] [General]
Hello! I was not able to find recipes / hyperparaneters for v2.1 local speaker segmentation model, is it possible to share it with community?

**Comment by FrenchKrab** [comment]
Not sure it fully answers your question but you can find the 2.1 version of the segmentation model here:
https://huggingface.co/pyannote/segmentation/tree/2.1

And you can refer to the tutorial notebooks in this repository to train it.

**Reply by vanIvan** [reply]
Got it, thank you for answering. Could you please also clarify which notebook to use:
* I found [this one](https://github.com/FrenchKrab/IS2023-powerset-diarization/blob/master/notebook_ps_diarization.ipynb) as separate repo, using pyannote itself, and seems like one of the latest work on training local speaker segmentation model
* Or [this notebook](https://github.com/pyannote/pyannote-audio/blob/develop/tutorials/training_a_model.ipynb) from pyannote

**Reply by FrenchKrab** [reply]
I think you're better off following the official pyannote notebook using the latest pyannote version.
The other notebook can also be a good reference but it isnt kept up to date with pyannote.
